.gitignore

```diff
@@ -4,3 +4,4 @@ node_modules
 .eslintcache
 package-lock.json
 yarn.lock
+**/.idea/
```

Comment:
Just use `.idea`

Reply:
It shouldn't be in this project `.gitignore`, because it's not related to stylelint directly. How to setup global `.gitignore`: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
🤷‍♂️ sure I'll remove it. Makes no difference to me